packages/opds-web-client/src/components/__tests__/BorrowButton-test.tsx

```diff
@@ -32,7 +32,7 @@ describe("BorrowButton", () => {
 
   it("shows button", () => {
     let button = wrapper.find("button");
-    expect(button.props().style).to.equal(style);
+    expect(stringify(button.props().style)).to.equal(stringify(style));
```

Comment:
alternately, you can probably use `deep.equal` instead of `equal`? That's probably better since things could end up in different orders in the strings.